qemu/tests/single_driver_install.py

```diff
@@ -138,6 +138,7 @@ def run(test, params, env):
 
         if params.get_boolean('need_destroy'):
             vm.destroy()
+            params = params.object_params(vm.name)
```

Comment:
Suggest `vm.create()` , it will use the `vm.params` as `params`, same with the pre vm totally.

Reply:
@yanan-fu  many thanks, update it

Reply:
> Suggest `vm.create()` , it will use the `vm.params` as `params`, same with the pre vm totally.

@yanan-fu  Many  Thanks, have updated it